h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -361,15 +265,11 @@ public void run() {
 
   @Override
   public void stop() {
-    if (null == jobs) return; // already stopped
-    for (Job j : jobs) j.stop();
-    for (Job j : jobs) j.get(); // Hold until they all completely stop.
-    jobs = null;
-
-    runCountdown.stop();
+    if (null == trainingStepsExecutor) return; // already stopped
+    trainingStepsExecutor.stop();
     eventLog().info(Stage.Workflow, "AutoML build stopped: " + EventLogEntry.dateTimeFormat.format(runCountdown.stop_time()))
             .setNamedValue("stop_epoch", runCountdown.stop_time(), EventLogEntry.epochFormat);
-    eventLog().info(Stage.Workflow, "AutoML build done: built " + modelCount + " models");
+    eventLog().info(Stage.Workflow, "AutoML build done: built " + trainingStepsExecutor.modelCount() + " models");
```

Comment:
`it is just to summarise what was logged along the execution`
I think it was the original purpose.
Each event is first logged during execution at its proper level (I remember this because I added automatic logging when calling `eventLog().info` for example).